hack/prepare-ec-credentials.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo "ecApiKey: '${EC_API_KEY}'" > ec-credentials.conf
```

Comment:
Is it expected to quote the value of EC_API_KEY?

Reply:
I do not think quoting is required. It looks like I can get rid of those?